cluster-autoscaler/OWNERS

```diff
@@ -1,9 +1,11 @@
 approvers:
-- fgrzadkowski
 - mwielgus
```

Comment:
mwielgus is already a top level owner, so probably redundant here (or I want to be included as well :) )

Reply:
Done